sdk-api-src/content/cfapi/ns-cfapi-cf_placeholder_create_info.md

```diff
@@ -56,7 +56,10 @@ Contains placeholder information for creating new placeholder files or directori
 
 ### -field RelativeFileName
 
-The name of the child placeholder file or directory to be created.
+The name of the child placeholder file or directory to be created. 
+
+This parameter must not contain a relative path or else CfCreatePlaceholders will fail with ERROR_CLOUD_FILE_INVALID_REQUEST.
```

Comment:
I think we should avoid making error codes contractual. I think we can simplify to

> The name of the child placeholder file or directory to be created. It should consist only of the file or directory name.
>
> For example, if the sync root of the provider is `C:\SyncRoot` then to create a placeholder named `placeholder.txt` in a subdirectory `SubDirectory` of the sync root, call the **CfCreatePlaceholders** function with _BaseDirectoryPath_ equal to `C:\SyncRoot\SubDirectory` and set the *RelativePathName* field of the **CF_PLACEHOLDER_CREATE_INFO** to `placeholder.txt`.